SDWebImage/UIButton+WebCache.m

```diff
@@ -146,18 +146,10 @@ - (void)sd_setBackgroundImageWithURL:(nullable NSURL *)url
                            completed:completedBlock];
 }
 
-- (void)sd_setImageLoadOperation:(id<SDWebImageOperation>)operation forState:(UIControlState)state {
```

Comment:
These two methods exist in SD 3.x, but is useless after SD 4.x refactory. (UIButton will use UIView+WebCache category add just pass the key)